pandas/core/arrays/interval.py

```diff
@@ -583,6 +583,8 @@ def _cmp_method(self, other, op):
                 result[i] = op(self[i], obj)
             except TypeError:
                 if obj is NA:
+                    # comparison returns NA, which we (for now?) treat like
```

Comment:
It's actually only for comparison of np.nan with pd.NA, comparison with Intervals is already (for now) returning False

Reply:
can you suggest an edit so i can just hit "commit" on it

Reply:
updated per request + green